Make slight corrections for the docs branch.

Kichura opened the pull request:
Resolves a few grammar errors in the docs and adds some other touches here-and-there because why not?

Kichura committed:
Use environment variable, Include fedora's DNF.

The environment variable is alot more cooler than using the lame username approach.

Kichura committed:
Redo tested systems list a bit.

cuz why not?

Kichura committed:
Correct "macOS" and "other" -> "others".

Kichura committed:
Fix "It's" word, Seperate "Vanilla" + "Installer" words.

Madis0 commented on README.md:
```diff
@@ -1,7 +1,7 @@
 <img alt="Built with Markdown" height="40" src="https://cdn.jsdelivr.net/npm/@intergrav/devins-badges@2/assets/compact/built-with/markdown_vector.svg">
 
-Welcome to the VanillaInstaller docs! In here, you can find useful information about this project and its development.
+Welcome to the Vanilla Installer docs! In here, you can find useful information about this project and it's development.
```
"its" was correct here

Kichura replied:
Darn.

Madis0 commented on credits.md:
```diff
@@ -275,6 +275,6 @@ SOFTWARE.
 * [https://docs.python.org/3.10/library/sys.html#sys.platform](https://docs.python.org/3.10/library/sys.html#sys.platform)
 
-### Other
+### Others
```
both are valid IMO

Kichura replied:
I chose "others" because the wording made morely sense.

Madis0 commented on installation.md:
```diff
@@ -25,7 +25,7 @@ It's recommended to install Python using [Homebrew](https://brew.sh), however in
 Because of Linux having many different package managers, we've decided to make the way to install on Linux via `pip`, or building from source using `python` and `poetry`, which isn't covered here.
 
-1. Install Python 3.8 or higher (3.11 recommended) from your package manager - for example `sudo apt install python3` or `sudo pacman -S python3`
+1. Install Python 3.8 or higher (3.11 recommended) from your package manager - for example `sudo apt install python3` or `sudo pacman -S python3` or `sudo dnf install python3` 
```
What's that?

Kichura replied:
This is fedora's default packaging command for installing python3.

Madis0 replied:
Well then put the distro names in parentheses (to make it useful) or remove the last two examples altogether (to only have Debian's serve as an example).

Kichura replied:
On it.

Kichura replied:
Not the best i could do in commit but feel free to send suggestions on how to make it better if needed.

Kichura committed:
Undo "it's" correction. (In request of Madis0's suggestion)

Kichura committed:
Correct grammar in changelogs readme.

Kichura committed:
Move the 3 python3 installers to new format.

Undoubtly not the kind of format madis0 wanted but this is what i could pull off.

Madis0 commented on changelogs/README.md:
```diff
@@ -1,6 +1,6 @@
 # Changelogs
 
-The changelogs for VanillaInstaller follow a similar format to [Iris](https://irisshaders.net)'s format, with a full changelog and a summary.
+The changelogs for Vanilla Installer follows a similar format to [Iris](https://irisshaders.net)' format, with a full changelog and a summary.
```
changelogs follow or changelog follows

Kichura replied:
*(ahem).*

Kichura committed:
*ahem*.

Madis0 reviewed (approved):
LGTM, though I leave the "Vanilla Installer"/"VanillaInstaller" naming decision to osfan

osfanbuff63 commented on for-devs.md:
```diff
@@ -27,10 +27,12 @@ This one might be obvious, but please keep in mind to ALWAYS `Ctrl+F` before ren
 
-- Linux → Debian → Ubuntu → PopOS:
-  - My main system, should work 100% fine
+- Debian → Ubuntu → Pop! OS:
```
should these have spaces next to the arrows? or is it just github mobile formatting

Kichura replied:
They use periods to let you figure out if there is a space there or not, In this case; It's intentional accessibility which i did not purposely add.

osfanbuff63 reviewed (commented):
Looks pretty good, just one quick comment

Honestly I think Vanilla Installer with the space looks a bit better, and it's used other places so this is the right move imo

osfanbuff63 reviewed (approved):
LGTM!
